### DISCUSSION TITLE
ImportError: cannot import name 'trame' from 'trame.widgets'

### DISCUSSION THREAD

**Discussion by kerim371** [Kitware/trame#586] [General] [answered]
Hi,

Trying to run [some examples](https://github.com/Kitware/trame/blob/master/examples/validation/core/00_state.py) and I get an error:
```python
from trame.widgets import html, trame

# THE ERROR:
ImportError: cannot import name 'trame' from 'trame.widgets'
```
It seems my `trame` installation misses some package.

**Comment by jourdain** [comment, marked as answer]
https://kitware.github.io/trame/guide/intro/widgets.html

**Reply by kerim371** [reply]
Thank you!

Now I see that some of the `trame.widgets.{name}` package are **not** built-in and we have to install them additionally.

For example in my case I needed to install [trame-components](https://github.com/Kitware/trame-components)
